puree/src/androidTest/java/com/cookpad/puree/outputs/PureeOutputTest.java

```diff
@@ -83,30 +81,38 @@ public String type() {
         };
         output.registerFilter(new PureeFilter() {
             @Override
-            public JsonObject apply(JsonObject jsonLog) {
-                JsonObject jsonObject = new JsonObject();
-                jsonObject.addProperty("filter1", "foo");
-                return jsonObject;
+            public String apply(String jsonLog) {
+                try {
+                    JSONObject jsonObject = new JSONObject();
+                    jsonObject.put("filter1", "foo");
+                    return jsonObject.toString();
+                } catch (JSONException e) {
+                    throw new RuntimeException(e);
```

Comment:
What's the reason for catching `JsonException` and re-throwing `RuntimeException` here? Also, the method's signature specifies it throws `JsonException`

Reply:
Because this is Java and that's a checked exception so it's forced me to re-trow with an unchecked exception 🤷‍♂️